src/components/profileview/NextProfile.tsx

```
@@ -0,0 +1,15 @@
+interface NextProfileProps {
```

ESLint

[error] 'NextProfileProps' is defined but never used.

(@typescript-eslint/no-unused-vars)